src/main/resources/resources/cohortdefinition/printfriendly/conceptSet.ftl

```diff
@@ -1,22 +1,22 @@
 <#import "./utils.ftl" as utils>
 <#list conceptSets as conceptSet>
-### ${conceptSet.name}
-<#list conceptSet.expression.items>
-|Concept ID|Concept Name|Code|Vocabulary|Excluded|Descendants|Mapped
-|:---|:---------------------------------------|:--|:-----|:--:|:--:|:--:|
-<#items as conceptSetItem>
-|${conceptSetItem.concept.conceptId?c}|<#--
+    ### ${conceptSet.name}
+    <#list conceptSet.expression.items>
+        |Concept ID|Concept Name|Code|Vocabulary|Excluded|Descendants|Mapped
+        |:---|:---------------------------------------|:--|:-----|:--:|:--:|:--:|
+        <#items as conceptSetItem>
+            |${conceptSetItem.concept.conceptId?c}|<#--
```

Comment:
You want to be extremly careful with changing the formatting of the Freemarker templates (ftl files) that are building the markdown format, since these are very sensitive to whitespacing.  Unless you need to change this, I would not touch any of these.